plugins/editorconfig/LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013 Kevin Bell
```

Comment:
We should keep the original copyright here, proper way would be to add JetBrains on a second line with proper year range

Reply:
Should it be
```
Copyright (c) 2013 Kevin Bell
Copyright (c) 2018 JetBrains
```
or 
```
Copyright (c) 2013 Kevin Bell,
2018 JetBrains
```
?